app/src/test/java/com/x8bit/bitwarden/data/auth/datasource/network/service/OrganizationServiceTest.kt

```diff
@@ -103,6 +111,27 @@ class OrganizationServiceTest : BaseServiceTest() {
         val result = organizationService.getOrganizationKeys("orgId")
         assertTrue(result.isFailure)
     }
+
+    @Suppress("MaxLineLength")
+    @Test
+    fun `getVerifiedOrganizationDomainSsoDetails when response is success should return valid response`() =
+    runTest {
```

Comment:
You should re-run the formatter here, all the indentation is off

Reply:
Sorry thought I had it running automatically. Fixed now. 👍